guides/common/modules/proc_promoting-your-content-view-to-the-new-lifecycle-environment.adoc

```diff
@@ -5,7 +5,7 @@ You can now promote your new Content View to the *Production* environment using
 
 .Procedure
 . In the {ProjectWebUI}, navigate to *Content* > *Content Views* > *Select your Content View*
-. On the *Versions* tab Click *Promote* for desired version under the Actions column.
+. On the *Versions* tab click *Promote* for desired version under the Actions column.
```

Comment:
> I am unsure about the whole "click vs select" thing, but if others approve it, then it's fine for me.
> 
> Do you by any chance have a reference/source for that?
> 
> Some suggestions, but I think this can get merged soon.

It appears to be "click" for buttons and "select" when choosing from a list of options.